openstf/pipeline/predict_sklearn.py

```diff
@@ -23,21 +24,9 @@ def predict_pipeline(pj):
         t_behind_days=14, t_ahead_days=3
     )
 
-    input_data = DataBase().get_model_input(
-        pid=pj["id"],
-        location=(pj["lat"], pj["lon"]),
-        # TODO it makes more sense to do the string converion in the
-        # DataBase class and use pure Python datatypes in the rest of the code
-        datetime_start=str(datetime_start),
-        datetime_end=str(datetime_end),
-    )
-
     # Get model
     model = joblib.load(MODEL_LOCATION / "model.sav")
```

Comment:
I guess this will change when @JanMaartenvanDoorn 's changes are merged. This current implementation is not very flexible, all models will be loaded from a single location.

Reply:
Should'nt we add `model `in the inputs of this function?

Reply:
The `joblib.load()` function takes a filename/path to a previously saved model file, so I think this is correct? 
![image](https://user-images.githubusercontent.com/6715707/118002717-2f72c500-b348-11eb-9642-26384329f1c8.png)

When dumping/saving you would indeed add the `model` as an argument -> `joblib.dump(model, model_save_path)`. Or am I missing something here?

Reply:
this is correct, however this functionality is now moved to the serializer and should be resolved when merging